src/variables.scss

```diff
@@ -0,0 +1,43 @@
+$lightest: hsla(200, 14%, 96%, 1);
```

Comment:
This is great! I know we have a lot of styled components, and using these might be tricky, but I'm excited to implement them down the road and it's nice to have them "ready to go"